docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +25,22 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To connect to Iceberg, you need:
```

Comment:
This is a bit misleading or poorly phrased, since Iceberg is table format, not a system that you connect to (unlike other systems like MySQL). I think it would be better to say "To use Iceberg, you need"

Reply:
Done.